pkg/controller/storagecluster/reconcile.go

```diff
@@ -113,21 +113,16 @@ func newCephCluster(sc *ocsv1alpha1.StorageCluster) *rookCephv1.CephCluster {
 				Enabled:        true,
 				RulesNamespace: "openshift-storage",
 			},
-			// TODO: Enable once StorageClassDeviceSet is merged into Rook
-			/*
-				Storage: rook.StorageScopeSpec{
-					StorageClassDeviceSets: newStorageClassDeviceSets(sc.StorageDeviceSet),
-					// XXX: Depending on how rook-ceph is going to use
-					// StorageClassDeviceSets we would be setting other required parameters
-					// for CephCluster.Storage
-				},
-			*/
+			Storage: rook.StorageScopeSpec{
+				StorageClassDeviceSets: newStorageClassDeviceSets(sc.StorageDeviceSet),
+				// XXX: Depending on how rook-ceph is going to use
```

Comment:
Now that https://github.com/rook/rook/pull/3451 is merged, do we need to update anything here?

Reply:
This comment is superfluous, please remove it.